### DISCUSSION TITLE
[Enhancement]: Audio crossfade

### DISCUSSION THREAD

**Discussion by PIndividual** [ciderapp/Cider#620] [Feature Request]
### The feature you would like

Allow songs to crossfade and enables users to set the seconds (usually goes from 1 to 12) it takes is a great feature on old iPods and Spotify, it brings a seamless experience.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by tva2000hn** [comment]
Very hard because of the way MusicKit JS works
